ObjectMapper/Core/Mapper.swift

```diff
@@ -412,38 +412,31 @@ extension Mapper where N: Hashable {
 }
 
 extension Dictionary {
-	internal func map<K: Hashable, V>(@noescape f: Element -> (K, V)) -> [K : V] {
```

Comment:
I have not measured the performance of reduce so I don't know if this is true. However I don't see a compelling reason to use it in this situation. I personally find the syntax is a bit clearer without it.